### PR TITLE
[LAB-14] Retrieve attached content for Assignment/Submission

### PR DESCRIPTION
**Pull request by Marko-Shevchuk**

### CODE REVIEW

File: src/main/java/com/pnu/lab/control/labcontrol/api/AssignmentAttachedContentController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    private final AssignmentAttachedContentService service;
+
+    @PostMapping
+    public AssignmentAttachedContent create(@Valid @RequestBody AssignmentAttachedContent assignmentAttachedContent) {
+        return service.create(assignmentAttachedContent);
+    }
+
+    @PutMapping
+    public AssignmentAttachedContent update(@Valid @RequestBody AssignmentAttachedContent assignmentAttachedContent) {
```
Comment (ArsenOOO7) [inline src/main/java/com/pnu/lab/control/labcontrol/api/AssignmentAttachedContentController.java:31]: U cannot update attached content, only create/delete

File: src/main/java/com/pnu/lab/control/labcontrol/api/SubmissionAttachedContentController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+public class SubmissionAttachedContentController {
+
+    private final SubmissionAttachedContentService service;
+
+    @PostMapping
+    public SubmissionAttachedContent create(@Valid @RequestBody SubmissionAttachedContent submissionAttachedContent) {
+        return service.create(submissionAttachedContent);
+    }
+
+    @PutMapping
```
Comment (ArsenOOO7) [inline src/main/java/com/pnu/lab/control/labcontrol/api/SubmissionAttachedContentController.java:30]: +1

**Review by ArsenOOO7 — APPROVED** [review]

**Review by JJerome-NM — CHANGES_REQUESTED** [review]
+1

**Review by ArsenOOO7 — CHANGES_REQUESTED** [review]
When u remove either Assignment or Submission, u should also delete attached content and comments

**Review by ArsenOOO7 — APPROVED** [review]

**Review by JJerome-NM — APPROVED** [review]
**_Request changes_**

**Pull request merged**